### DISCUSSION TITLE
Identifying updated records using API [Aug 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3190] [Discord Threads]
> 🙋‍♂️ **ccoli**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Webhook is ideal way. If you wish to do it using API, it can be done via polling periodically & conditionally retrieving records from a table based on `updated-at` timestamp range.  
#  
> 🙋‍♂️ **ccoli**  
#### thank you <@857998597474484295>  for ur reply. but polling is a bit different that I wish to do.   
####    
#### the thing I wish to do is the below.   
#### "deal" table has "cpid", "subject" columns and some defaults columns as well.   
####    
#### 1. update cpid column (cpid is just an example column) on nocodb   
####    
#### 2. fetch some data with cpid   
####    
#### 3. update the record with data fetched on subject.   
####    
#### without using webhooks, so far is there any way ??   
####    
#### I'm justing thinking this way can u tell me if this ok or not??   
####    
#### After updating record, there is an post api with payload.   
#### so on the routing, I catch the payload and somehow call the api to fetch the subject data.   
#### and with the subject data. I update the record.   
####    
#### but I'm not sure if this is possible and ok to do it.  
#  
> 👨‍🦲 **dstala**  
#### without using webhook, this would become tedious I believe.    
####    
#### > so on the routing, I catch the payload and somehow call the api to fetch the subject data.   
#### > and with the subject data. I update the record.   
####    
#### So if you are manually adding a patch for a particular table, it might work as long as the API invocation parameters are correct. not recommended though - as it would be pretty much a hack;  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1006094169942528060)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
